i18n/de_DE/mail/donation_confirmation/moderation_other_reason.twig

```diff
@@ -0,0 +1,7 @@
+
+TODO:
+Hinweis über die eingeleitete Moderation (anderer Grund als zu hoher Betrag) (DE)
+
+verfügbare Text-Parameter:
+{$ formatted_amount $}
+{$ interval $}
```

Comment:
I think you can put the contents of https://github.com/wmde/fundraising-frontend-content/blob/test/i18n/de_DE/mail/donation_confirmation/needs_moderation.twig in this file.